config/tests/testgrids/config_test.go

```diff
@@ -330,12 +330,17 @@ func TestConfig(t *testing.T) {
 
 		// Dashboards that match this dashboard group's prefix should be a part of it
 		for dashboard := range dashboardmap {
-			if strings.HasPrefix(dashboard, dashboardGroup.Name+"-") {
-				group, ok := tabs[dashboard]
+			thisGroup := dashboardGroup.Name
+			if strings.HasPrefix(dashboard, thisGroup+"-") {
+				assignedGroup, ok := tabs[dashboard]
 				if !ok {
-					t.Errorf("Dashboard %v should be in dashboard_group %v", dashboard, dashboardGroup.Name)
```

Comment:
The error was saw was:
```
Dashboard knative-sandbox-eventing-prometheus should be in dashboard_group knative
```
It's actually from this line right?

Reply:
Good catch.